.github/workflows/initialize-repo.yml

```diff
@@ -22,6 +22,7 @@ jobs:
           if [[ "${modulename}" != "kickstart" ]]; then
             git mv imageroot/systemd/user/kickstart.service imageroot/systemd/user/${modulename}.service
             sed -i "s/kickstart/${modulename}/g" $(find * -type f)
+            git rm -f .github/workflows/initialize-repo.yml
```

Comment:
Sadly (or luckily) GitHub actions does not allow to modify workflow files with the default security token... I didn't try this specific command however: maybe a removal is allowed...